tpm2/constants.go

```diff
@@ -78,6 +78,13 @@ const (
 	SessionTrial  SessionType = 0x03
 )
 
+// SessionAttributes represents an attribute of a session.
+type SessionAttributes byte
+
+const (
+	AttrContinueSession = SessionAttributes(1)
```

Comment:
Add the rest of attributes from the spec

Reply:
done